src/ctimer/mod.rs

```diff
@@ -0,0 +1,49 @@
+//! API for the CTimer peripheral
```

Comment:
Small nit remaining from a15e2a0ec9f8ca917485d0f2403416f6e8c187b7, this should probably also be `CTIMER`

Reply:
Pushed a commit addressing that.